services/route_manager/src/main/java/com/futurewei/alcor/route/service/Impl/NeutronRouterServiceImpl.java

```diff
@@ -469,6 +466,151 @@ public ConnectedSubnetsWebResponse getConnectedSubnets(String projectId, String
         return connectedSubnetsWebResponse;
     }
 
+    @Override
+    public InternalRouterInfo updateRoutingRule (String routerId, String subnetId, RoutesToNeutronRouterRequestObject routerObject) throws ResourceNotFoundException, ResourcePersistenceException, RouterUnavailable, RouterTableNotExist, DestinationOrNexthopCanNotBeNull, OwnerInNeutronRouteTableNotFound {
+        List<InternalRoutingRule> updateRoutes = new ArrayList<>();
+
```

Comment:
Chane `RoutesToNeutronRouterRequestObject routerObject` to a more generic routes object like we discuss before.

Reply:
Sure